nginx-proxy.conf

```diff
@@ -17,3 +17,5 @@ add_header 'Access-Control-Allow-Origin' '*';
 
 # Required for uploading files on various apps
 client_max_body_size 500m;
+
+proxy_read_timeout 200s;
```

Comment:
Is this the amount of time that compiling assets takes? 😱

Reply:
The default is 60s and it times out. I followed a scientific formula of adding a zero to the end and not seeing 504s at the moment. Which is a huge relief.

Reply:
Sorry for the delayed comment. Could we add a comment to indicate the reason behind this setting, like the other lines in this file? Although it's in the commit, that information become less visible if someone decides to reorder the lines in this file; and it's just helpful to read it in-place, as the meaning isn't going to change.